Content.Client/GameObjects/Components/Suspicion/TraitorOverlay.cs

```diff
@@ -33,7 +33,6 @@ public class TraitorOverlay : Overlay
             IEyeManager eyeManager)
             : base(nameof(TraitorOverlay))
         {
-            _entityManager = entityManager;
```

Comment:
Why did you remove this? The game will crash when the overlay is applied.

Reply:
Sorry, my bad. I used a script that removes lines with `_entityManager` to delete all `private readonly IEntityManager _entityManager = default!;` lines. But I think that some other lines was affected as well.